cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
+yours are different!
+
+However, the plan is to include the following
++ Nomenclature (multiple languages)
++ [Types of Cloth Diapers](#types-of-cloth-diapers)
++ [Materials and Absorbency](#materials-and-absorbency)
++ [Newborn vs One Size](#newborn-vs-one-size)
++ Velcro vs Poppers
++ [Our Choices](#our-choices)
++ Things to Know
++ What causes leaks
+
+## Types of Cloth Diapers
+![image](https://user-images.githubusercontent.com/4185328/169092681-4b44aaef-1127-4937-b906-1b2ddc864821.png)
+
+I don't want to go into all the different types of cloth diaper just yet as
```

Comment:
"I don't want ..."

Is your plan for this to be written in the "voice" of a mother (_in this case your own voice_),
or do you plan to have a more collective voice and thus more inclusive to the both the reader 
and inviting more contributors? e.g:

```suggestion
We won't go into all the different types of cloth diaper just yet as
```